sdk/tests/deqp/functional/gles3/es3fMultisampleTests.js

```diff
@@ -212,6 +212,51 @@ goog.scope(function() {
     /** Copy the constructor */
     es3fMultisampleTests.MultisampleCase.prototype.constructor = es3fMultisampleTests.MultisampleCase;
 
+    /* Rest states */
+    es3fMultisampleTests.MultisampleCase.prototype.deinit = function() {
+        gl.colorMask(true, true, true, true);
+        gl.depthMask(true);
+
+        gl.clearColor(0.0, 0.0, 0.0, 0.0);
+        gl.clearDepth(1.0);
+        gl.clearStencil(0);
+
+        gl.disable(gl.STENCIL_TEST);
+        gl.disable(gl.DEPTH_TEST);
+        gl.disable(gl.BLEND)
+        gl.disable(gl.SAMPLE_COVERAGE);
+        gl.disable(gl.SAMPLE_ALPHA_TO_COVERAGE);
```

Comment:
In TestCaseWrapper::deinit, glu::resetState() will do this cleanup at the end of each test.
https://android.googlesource.com/platform/external/deqp/+/master/modules/gles3/tes3TestPackage.cpp#69
https://android.googlesource.com/platform/external/deqp/+/master/framework/opengl/gluStateReset.cpp#35